### PR TITLE
Add new config option to polls

### LINKED ISSUE #4721: New poll option - permanent vote
Once you have voted, you have voted and you cannot change your vote.

### PR DESCRIPTION
**Pull request by julianjelfs**
fixes #4721

### CODE REVIEW

File: frontend/app/src/i18n/en.json [hunk 1]
```diff
@@ -391,6 +391,7 @@
         "oneDay": "1 Day",
         "oneWeek": "1 Week",
         "allowMultipleVotes": "Allow multiple votes",
+        "allowChangeVotes": "Allow user to change vote",
```
Comment (hpeebles) [inline frontend/app/src/i18n/en.json:394]: "Allow users* ..." is probably better

File: frontend/app/src/components/home/PollContent.svelte [hunk 2]
```diff
@@ -24,13 +24,16 @@
 
     $: numberOfVotes = totalVotes(content);
 
+    $: cannotVote =
+        content.ended || readonly || (haveIVoted && !content.config.allowUserToChangeVote);
```
Comment (hpeebles) [inline frontend/app/src/components/home/PollContent.svelte:28]: If the config is set to allow multiple then they can still vote on other options.

Comment (julianjelfs) [reply frontend/app/src/components/home/PollContent.svelte:28]: yes I thought it made sense that this new setting only applied for single vote polls

Comment (julianjelfs) [reply frontend/app/src/components/home/PollContent.svelte:28]: Oh wait - I see what you mean

Comment (julianjelfs) [reply frontend/app/src/components/home/PollContent.svelte:28]: although - if allow multiple is true, allow user to change will also always be true

**Review by hpeebles — APPROVED** [review]

**Pull request merged**